src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+    Collapse,
+    Container,
+    Nav,
+    Navbar,
+    NavbarToggler,
+    NavItem
+} from 'reactstrap';
+import { NavLink } from 'react-router-dom';
+import Profile from './Profile';
+
+interface NavBarProps {}
+
+const NavBar: React.FunctionComponent<NavBarProps> = ({}) => {
```

Comment:
If this component doesn't take any props, there's no need for an empty object de-structuring here. Also, while it's arguably here to guide in the right direction for future use cases, I don't see a need to provide an empty props interface.